src/components/TinderStyleVoting/PostInfoComponents/CardPostHeading.tsx

```
@@ -247,18 +248,51 @@
+															proposalCreatedAt={created_at as any}
+															timeline={timeline || []}
+															usedInPostPage
+														/>
+													);
+												})}
+											</div>
+										}
+									>
+										<div className='mt-0.5 cursor-pointer text-xs text-[#407BFF]'> & {post?.beneficiaries?.length - 1} more</div>
+									</Popover>
+								</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)